timeservice.go

```diff
@@ -293,10 +368,13 @@ func loadNTSKEConfig(log *zap.Logger, configFile string) *tls.Config {
 	}
 }
 
-func runServer(configFile, daemonAddr string, localAddr *snet.UDPAddr) {
+func runServer(configFile string) {
 	ctx := context.Background()
 
-	refClocks, netClocks := loadConfig(ctx, log, configFile, daemonAddr, localAddr)
+	cfg := loadConfig(log, configFile)
+	localAddr := localAddress(log, cfg)
+	daemonAddr := cfg.DaemonAddress
```

Comment:
Add a ( trivial) function `deamonAddresss(log *zap.Logger, cfg svcConfig)...`, just for consistency?

Reply:
Tough one. But yes, I think an accessor function would look nicer and more consistent.

Reply:
Done